modules/kvantum/hm.nix

```diff
@@ -0,0 +1,70 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: {
+  options.stylix.targets.kvantum = {
```

Comment:
Kvantum is more of an implementatiion detail rather than something which is being themed. Would it be sensible to either combine this with the existing KDE target, or rename it to QT since it presumably has an effect outside of KDE too?

Reply:
I renamed the target to QT. I agree it makes more sense, KDE should contain only KDE specific integrations.